CmdStarter.Tests.Commands/Erroneous/AutoComplete/NonGenericNullCompletion.cs

```diff
@@ -0,0 +1,14 @@
+﻿using com.cyberinternauts.csharp.CmdStarter.Tests.Commands.Interfaces;
+using NUnit.Framework;
```

Comment:
The `NUnit` framework was never used in the commands because I thought of them as data, not tests.

I like what you did. I am only questioning myself. Shall we follow the principle above? If so, maybe I did a mistake by adding this package in `Tests.Common` project.

If we agree on what we will do, we could do this task in the CMD-39. I could add to remove this package from `Tests.Common`, move the attributes `TestCase*` to `Tests` project, move interfaces in `Tests` to `Tests.Common` and fix errors.

Example:
`TestDelegate` could be replaced by a `Delegate` that would be called in the `Assert` this way: `Assert.Throws(instance.TypeOfException, () => t.ErrorRunner2.DynamicInvoke());` OR something similar.

Reply:
Hmm.. Perhaps we should keep that principle in the heart of separation of concerns.

I only used `TestDelegate` to save the hassle of doing that ugly invoke in the `Assert`.

Reply:
I think it would a great idea to tend toward this principle.

Reply:
Done.